azure/ConsiderationsForServiceDesign.md

```diff
@@ -332,74 +406,77 @@ HTTP/1.1 200 OK
-The response may also include an `Operation-Location` header for backward compatibility.
-If the resource supports ETags, the response may contain an `etag` header and possibly an `etag` property in the resource.
+3. After waiting at least the amount of time specified by the previous response's `Retry-after` header,
+the client issues a GET request to the status monitor using the ID in the body of the initial response.
+The GET operation for the status monitor is documented in the REST API definition and the ID
+is the last URL path segment.
 
-```text
-HTTP/1.1 201 Created
-Operation-Id: 22
-Operation-Location: https://items/operations/22
-etag: "123abc"
+4. The status monitor responds with information about the operation including its current status,
+which should be represented as one of a fixed set of string values in a field named `status`.
+If the operation is still being processed, the status field will contain a "non-terminal" value, like `NotStarted` or `Running`.
 
-{
-  "id": "FooBar",
-  "etag": "123abc",
-  "prop1": 555,
-  "prop2": "something"
-}
-```
+5. After the operation processing completes, a GET request to the status monitor returns the status monitor with a status field set to a terminal value -- `Succeeded`, `Failed`, or `Canceled` -- that indicates the result of the operation.
+If the status is `Failed`, the status monitor resource contains an `error` field with a `code` and `message` that describes the failure.
+If the status is `Succeeded`, the operation results will be returned in the `result` field of the status monitor.
 
-The client will issue a GET to the status monitor to obtain the status of the operation performing the additional processing.
+6. There may be some cases where a long-running action operation can be completed before the response to the initial request.
+In these cases, the operation should still return a `202 Accepted` with the `status` property set to the appropriate terminal state.
 
-```text
-GET https://items/operations/22?api-version=2022-05-01
-```
+7. The service is responsible for purging the status-monitor resource.
+It should auto-purge the status monitor resource after completion (at least 24 hours).
+The service may offer DELETE of the status monitor resource due to GDPR/privacy.
 
-When the additional processing completes, the status monitor will indicate if it succeeded or failed.
+### Long-running action operation not related to a resource
 
-```text
-HTTP/1.1 200 OK
+When an long-running action operation is not related to a specific resource (a batch operation is one example),
+another approach is needed.
 
-{
-   "id": "22",
-   "status": "Succeeded"
-}
-```
+A long-running action operation not related to a resource should use a PUT operation for a resource that combines a status monitor with the request and results of the operation. In this pattern, clients obtain the status of the operation by issuing a GET on the same URL as the PUT operation.
```

Comment:
This doesn't make sense. An LRO action operation *not related to a resource* should use a PUT operation *for a resource*...

You just said it isn't related to a resource. So how I can I do a PUT to a resource? How is that resource associated?

Reply:
Agreed. I have revised the text here to eliminate this confusion (and hopefully not create any new confusion).